packages/common/src/utils/analytics/google.ts

```diff
@@ -1,13 +0,0 @@
-import { global } from './utils';
-
-export const track = (_, data) => {
-  if (typeof global.ga !== 'undefined') {
-    global.ga('send', data);
```

Comment:
We'll lose Google Analytics from this, but that should be fine, right?

Reply:
Yes, my understanding is that we want to get rid of GA anyway, for privacy concerns